Makefile

```diff
@@ -117,7 +117,7 @@ test-pydocstyle:
 .PHONY: test-pylint
 test-pylint:
 	pylint rockcraft
-	pylint tests --disable=invalid-name,missing-module-docstring,missing-function-docstring,redefined-outer-name,too-many-arguments,too-many-public-methods
+	pylint tests --disable=invalid-name,missing-module-docstring,missing-function-docstring,redefined-outer-name,too-many-arguments,too-many-public-methods,no-member
```

Comment:
I had to add this one because `pylint` keeps complaining that `Project` doesn't have members that it clearly has

Reply:
What version of pylint are you using, and which version of Python is it running under?

Reply:
pylint==2.15.8, pylint-pytest==1.0.3, Python 3.8 I think (from focal)